README.md

```diff
@@ -851,6 +847,17 @@ ANSWERS.addComponent('VerticalResults', {
     // Optional, whether to display all results in the vertical when no results are found. Defaults to false, in which case only the no results card will be shown.
     displayAllResults: false
   },
+
+  /**
+   * DEPRECATED: The config below is DEPRECATED.
```

Comment:
Thoughts on "The config options below are DEPRECATED" to make it clearer that it is multiple options rather than a single one?

Reply:
updated!